api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -97,39 +101,51 @@ private LeonardoCreateRuntimeRequest buildCreateRuntimeRequest(
-        .addScopesItem("https://www.googleapis.com/auth/userinfo.email")
-        .addScopesItem("https://www.googleapis.com/auth/userinfo.profile")
-        .runtimeConfig(
-            new LeonardoMachineConfig()
-                .cloudService(CloudServiceEnum.DATAPROC)
-                .masterDiskSize(
-                    Optional.ofNullable(clusterOverride.masterDiskSize)
-                        .orElse(config.firecloud.notebookRuntimeDefaultDiskSizeGb))
-                .masterMachineType(
-                    Optional.ofNullable(clusterOverride.machineType)
-                        .orElse(config.firecloud.notebookRuntimeDefaultMachineType)))
-        .toolDockerImage(workbenchConfigProvider.get().firecloud.jupyterDockerImage)
-        .welderDockerImage(workbenchConfigProvider.get().firecloud.welderDockerImage)
-        .customEnvironmentVariables(customEnvironmentVariables);
+    LeonardoCreateRuntimeRequest request =
+        new LeonardoCreateRuntimeRequest()
+            .labels(
+                ImmutableMap.of(
+                    RUNTIME_LABEL_AOU,
+                    "true",
+                    RUNTIME_LABEL_CREATED_BY,
+                    userEmail,
+                    RUNTIME_LABEL_AOU_CONFIG,
+                    RuntimeConfigurationType.USEROVERRIDE.equals(runtime.getConfigurationType())
+                        ? "user-override"
+                        : "default"))
+            .defaultClientId(config.server.oauthClientId)
+            // Note: Filenames must be kept in sync with files in api/src/main/webapp/static.
+            .jupyterUserScriptUri(assetsBaseUrl + "/initialize_notebook_runtime.sh")
+            .jupyterStartUserScriptUri(assetsBaseUrl + "/start_notebook_runtime.sh")
+            .userJupyterExtensionConfig(
+                new LeonardoUserJupyterExtensionConfig().nbExtensions(nbExtensions))
+            // Matches Terra UI's scopes, see RW-3531 for rationale.
+            .addScopesItem("https://www.googleapis.com/auth/cloud-platform")
+            .addScopesItem("https://www.googleapis.com/auth/userinfo.email")
+            .addScopesItem("https://www.googleapis.com/auth/userinfo.profile")
+            .runtimeConfig(
+                new LeonardoMachineConfig()
+                    .cloudService(CloudServiceEnum.DATAPROC)
+                    .masterDiskSize(
+                        Optional.ofNullable(clusterOverride.masterDiskSize)
+                            .orElse(config.firecloud.notebookRuntimeDefaultDiskSizeGb))
+                    .masterMachineType(
+                        Optional.ofNullable(clusterOverride.machineType)
+                            .orElse(config.firecloud.notebookRuntimeDefaultMachineType)))
+            .toolDockerImage(workbenchConfigProvider.get().firecloud.jupyterDockerImage)
+            .welderDockerImage(workbenchConfigProvider.get().firecloud.welderDockerImage)
+            .customEnvironmentVariables(customEnvironmentVariables);
+
+    return request;
```

Comment:
nit: perhaps this was changed for debugging, but definitely prefer how this was previously written - assigning to a variable only to immediately return it on the next line is a bit of a pet peeve of mine